packages/server/api/webscrape.py

```diff
@@ -1,20 +1,40 @@
 from http.server import BaseHTTPRequestHandler
+import urllib.request as request
+import re
 from urllib.parse import parse_qs
+from bs4 import BeautifulSoup
 
 class handler(BaseHTTPRequestHandler):
-
+  whitelisted_url_regexes = [
+    re.compile("^https:\/\/www.bonappetit.com")
+  ]
   def do_GET(self):
     queryString = self.path.split('webscrape?')
```

Comment:
I know this isn't part of this PR, but variables in Python should be snake cased, i.e. `query_string` or event `querystring`